multiversx_sdk_cli/docker.py

```diff
@@ -48,6 +49,9 @@ def run_docker(
     docker_args += ["--user", f"{str(os.getuid())}:{str(os.getgid())}"]
     docker_args += ["--rm", image]
 
+    if not(no_default_platform):
+        docker_args += ["--platform", "linux/amd64"]
```

Comment:
I am unaware if this parameter influences Docker at the same level as `DOCKER_DEFAULT_PLATFORM`.

Reply:
https://docs.docker.com/engine/reference/commandline/cli/#environment-variables
> DOCKER_DEFAULT_PLATFORM | Default platform for commands that take the --platform flag.

Reply:
But I have not tested on my computer in order to be 100% sure

Reply:
Also there is this one: https://stackoverflow.com/questions/65612411/forcing-docker-to-use-linux-amd64-platform-by-default-on-macos